### DISCUSSION TITLE
トラックパッド無し基板の右から2, 3列目のキーが反応しない

### DISCUSSION THREAD

**Discussion by sota1235** [Yowkees/keyball#325] [Q&A] [answered]
トラックパッド無しの基盤を左で半田付けしています
右から2, 3列目(親指キー含む)のキーが全て反応しません。親指キーはロープロ仕様になっています。
他のキーは反応してる状態です(REMAPのテストで認識できる)
※ 画像だと一番下の左から4番目が反応してないんですがこれは半田付けをし直して直っています

<img width="697" alt="image" src="https://github.com/Yowkees/keyball/assets/2968860/000bcccf-ea2a-4d39-ad0b-55d643e2bf95">

過去のdiscussionを見直したり半田付けが怪しそうな場所をやり直したりしたのですが電子工作初心者で、打てる手を打ち切って困ってるので起票しました…！
テスターは購入したのでそれでの切り分けはできます(ProMicroの半田付けがすぐ隣の穴とテンプラしてないことだけ確認済みです)。

以下が半田付けの写真です

![image](https://github.com/Yowkees/keyball/assets/2968860/60dc98f3-371a-45d7-b290-ec64f8055c32)

![image](https://github.com/Yowkees/keyball/assets/2968860/631020e3-bd3f-4dae-8ce0-0c4b1e649341)

お手数ですがどう対応していけば良いかご教授いただきたいです…！

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。
縦一列すべて入力できないようですので、下記画像の丸で示したダイオード２つの極性や半田付けを見直してみてください。
また、ProMicro裏に隣とショートしている端子がないかもご確認お願いします。

![KB61_COL12fix](https://github.com/Yowkees/keyball/assets/84209772/11bd1ea4-a8b3-42dc-aaf5-c0a05f7bfbdf)

**Reply by sota1235** [reply]
指摘していただいた箇所を半田付けしなおしたらキーが反応しました！ありがとうございます。
(全部のキーが反応したわけではないんですが少なくとも一列全て、という状況は回避できました。まだ反応しないキーに関しては別途Q&A起票させてください 🙇 )
